src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/Utility.cs

```diff
@@ -311,12 +311,12 @@ internal static Object GetSqlCipherMetadata(ushort ordinal, byte cipherAlgorithm
 
         internal static byte[] DecryptWithKey(byte[] cipherText, Object cipherMd, string serverName)
         {
-            return (byte[])SqlSecurityUtilDecryptWithKey.Invoke(null, new Object[] { cipherText, cipherMd, serverName });
+            return (byte[])SqlSecurityUtilDecryptWithKey.Invoke(null, new Object[] { cipherText, cipherMd, new SqlConnection() });
         }
 
         internal static byte[] EncryptWithKey(byte[] plainText, Object cipherMd, string serverName)
         {
-            return (byte[])SqlSecurityUtilEncryptWithKey.Invoke(null, new Object[] { plainText, cipherMd, serverName });
+            return (byte[])SqlSecurityUtilEncryptWithKey.Invoke(null, new Object[] { plainText, cipherMd, new SqlConnection() });
```

Comment:
Since the serverName is not used, it's better to remove it from the parameter list.